### PR TITLE
Added initial travis configuration and Dangerfile

### LINKED ISSUE #8: Setup travis.ci

### PR DESCRIPTION
**Pull request by brototyp**
This PR adds a travis configuration and a Dangerfile and thus closes #8.

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+has_readme_updates = git.modified_files.include?("README.md")
+if has_pod_changes && !has_readme_updates && !declared_trivial
+  warn("Are there any changes that should be explained in the `README.md`?")
+end
+
+
+# Perform swiftlint and comment violations inline
+swiftlint.config_file = '.swiftlint.yml'
+swiftlint.verbose = true
+swiftlint.lint_files inline_mode: true
```
Comment (brototyp) [inline Dangerfile:25]: This will execute swiftlint and comment the violations in the diff.

File: Dangerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+declared_trivial = github.pr_title.include? "#trivial"
```
Comment (pstued) [inline Dangerfile:3]: how about mention that in our contributing.md ?

File: Dangerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+has_pod_changes = !git.modified_files.grep(/MatomoTracker/).empty?
```
Comment (pstued) [inline Dangerfile]: MatomoTracker 🤔 🙃

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+language: objective-c
+osx_image: xcode10.1
+
+before_install:
+  - 'echo ''gem: --no-ri --no-rdoc'' > ~/.gemrc'
```
Comment (pstued) [inline .travis.yml:5]: Do we always want to override the ~/.gemrc ?

**Review by pstued — APPROVED** [review]

File: Dangerfile (hunk 2 above)
Comment (brototyp) [reply Dangerfile:3]: I think this is a quite minute detail. If you don't know it, nothing bad happens. I think I would not mention it somewhere. Maybe we could add it to the message that Danger will post? Something like: "You can prevent this message by declaring a PR `#trivial` in the title" or such.

File: Dangerfile (hunk 3 above)
Comment (brototyp) [reply Dangerfile]: Uhm ... *cough cough* ... you got me there 😊

File: .travis.yml (hunk 4 above)
Comment (brototyp) [reply .travis.yml:5]: That just happens on travis. Do you think there is an issue with that?

File: Dangerfile (hunk 2 above)
Comment (brototyp) [reply Dangerfile:3]: I added such a message.

File: Dangerfile (hunk 3 above)
Comment (brototyp) [reply Dangerfile]: Fixed this.

File: .travis.yml (hunk 4 above)
Comment (pstued) [reply .travis.yml:5]: I‘m just not sure whoelse mau write to gemrc. Let‘s keep it in mind and leave it like this

Comment (brototyp) [reply .travis.yml:5]: Since this is only executed on the travis CI machine I guess we are pretty safe here. But yes, let's keep this in mind.

**Pull request merged**